### PR TITLE
Swift Package Manager Integration

### PR DESCRIPTION
**Pull request by jjb3rd**
Moved files around according to Swift Package Manager guidelines and added Package.swift for integration with Swift Package Manager.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import PackageDescription
+
+let package = Package(
+  name: "JWT",
+  dependencies: [
+    .Package(url: "https://github.com/krzyzanowskim/CryptoSwift", versions: Version(0,2,2)..<Version(1,0,0))
```
Comment (kylef) [inline Package.swift]: We're using and testing against [CryptoSwift 0.1.1 in the podspec](https://github.com/kylef/JSONWebToken.swift/blob/master/JSONWebToken.podspec#L14), would you be able to update the pod spec to use `~> 0.2.0` so it's the same here?

---

As per semver, with versions less than 0, any minor updates can cause API changes and thus we shouldn't allow versions 0.2.2 to 1.0.0. Can we change this version range to the following so upstream changes won't break JWT:

```swift
Version(0, 2, 2) ..< Version(0, 3, 0)
```

**Comment by kylef** [comment]
Thanks for the pull request @jjb3rd, just that minor comment and also could you update the `source_files` inside the pod spec so that the travis CI build still passes?

After those changes could you squash your comments and then this is good to go :ok_hand:.

**Comment by jjb3rd** [comment]
How do I squash the comments?

**Comment by kylef** [comment]
@jjb3rd Check https://github.com/getpelican/pelican/wiki/Git-Tips#squashing-commits

**Pull request closed without merging**